build/Dockerfile-go1.20.6-solc0.8.13-ubuntu

```diff
@@ -37,4 +37,4 @@ COPY --from=go_builder /usr/local/go /usr/local
 ENV GOPATH /go
 ENV PATH $GOPATH/bin:/usr/local/go/bin:$PATH
 RUN mkdir -p "$GOPATH/src" "$GOPATH/bin" && chmod -R 777 "$GOPATH"
-RUN curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.52.0
\ No newline at end of file
+RUN curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.61.0
```

Comment:
I think docker image needs to be updated. @Sotatek-TinnNguyen Could you take a look and update the docker image if needed?

Reply:
@yoomee1313 which version of lint should I upgrade to?

Reply:
@Sotatek-TinnNguyen According to the latest build_base image, the installed linter version is v1.57.0. Can we update to v1.61.0?

Reply:
yes, let me update this